clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -206,6 +206,20 @@
       In: ${arg}.cu
       Out: ${arg}.${rewrite_extention_name}
 
+# ${CUDA_LIBRARIES} is cmake reserved variable for static version of the CUDA runtime library.
+# In SYCL side, SYCL runtime is default auto-loaded when option "-fsycl" is specified, so we remove it in SYCL side.
+- Rule: rule_user_defined_function_cuda_libraries
+  Kind: CMakeRule
+  Priority: Fallback
+  MatchMode: Partial
+  CmakeSyntax: cuda_files_in_user_defined_function_cuda_libraries
```

Comment:
can be changed to a short and simple name?

Reply:
okay, will update it